src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,61 @@
+package calculator;
+
+import calculator.domain.Operation;
+import calculator.service.OperationFactory;
+import calculator.view.InputView;
+import calculator.view.ResultView;
+
+public class Calculator {
+
+    public static void main(String[] args) {
+        Calculator calculator = new Calculator();
+
+        ResultView.print(calculator.calculate(InputView.input()));
+    }
+
+    public int calculate(String input) {
+
+        String[] split = validateData(input.split(" "));
+
+        // 홀수는 -> 숫자 /// 짝수는 연산
+        int num1 = validateInt(split[0]);
```

Comment:
```suggestion
        int num1 = getFirstOperand(split); // 예시
```

무엇을 위해 유효성 체크를 하는지 바로 파악하기가 힘들 것 같아요
적절한 이름을 부여해 보면 어떨까요?